src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -84,6 +85,13 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
             CurrentConfiguration = configurationProvider.GetConfiguration();
 
             SetBoundSolutionUIContext();
+
+            gitEventsMonitor.HeadChanged += GitEventsMonitor_HeadChanged;
```

Comment:
The class has a `Dispose` method that unhooks the other extension handlers. I'm not sure it's strictly necessary, but we should be consistent and unhook this one too.